src/openrct2/actions/TrackSetBrakeSpeedAction.cpp

```diff
@@ -57,7 +57,7 @@ GameActions::Result TrackSetBrakeSpeedAction::QueryExecute(bool isExecuting) con
 
     if (!LocationValid(_loc))
     {
-        return GameActions::Result(GameActions::Status::NotOwned, STR_NONE, STR_NONE);
+        return GameActions::Result(GameActions::Status::NotOwned, STR_NONE, STR_LAND_NOT_OWNED_BY_PARK);
```

Comment:
I think `STR_CANT_CHANGE_THIS` would be a good fit for the error title, rather than STR_NONE.

```suggestion
        return GameActions::Result(GameActions::Status::NotOwned, STR_CANT_CHANGE_THIS, STR_LAND_NOT_OWNED_BY_PARK);
```